src/multicellular_stage/MulticellularLayoutHelpers.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Conversion helpers between the full (gameplay) and editor layouts of a multicellular species
+/// </summary>
+public static class MulticellularLayoutHelpers
+{
+    public static void UpdateGameplayLayout(CellLayout<CellTemplate> target,
+        IndividualHexLayout<CellTemplate> copySourceTo, IndividualHexLayout<CellTemplate> source,
+        List<Hex> hexTemporaryMemory, List<Hex> hexTemporaryMemory2)
+    {
+        copySourceTo.Clear();
```

Comment:
maybe rename those to targetGameplayLayout and targetEditorLayout for more clarity (if that's what those are)